pkg/crypto/measure.go

```diff
@@ -8,29 +8,34 @@ import (
 )
 
 const (
-	// Blob type in PCR 7
-	Blob uint32 = 7
-	// BootConfig type in PCR 8
-	BootConfig uint32 = 8
-	// ConfigData type in PCR 8
-	ConfigData uint32 = 8
-	// NvramVars type in PCR 9
-	NvramVars uint32 = 9
+	// BlobPCR type in PCR 7
+	BlobPCR uint32 = 7
+	// BootConfigPCR type in PCR 8
+	BootConfigPCR uint32 = 8
+	// ConfigDataPCR type in PCR 8
+	ConfigDataPCR uint32 = 8
+	// NvramVarsPCR type in PCR 9
+	NvramVarsPCR uint32 = 9
 )
 
 // TryMeasureBootConfig measures bootconfig contents
-func TryMeasureBootConfig(name, kernel, initramfs, kernelArgs, deviceTree string) {
+func TryMeasureBootConfig(name, kernel, initramfs, kernelArgs, deviceTree, multiboot, multibootArgs string, modules []string) {
```

Comment:
Is this function still used now that `Boot()` does not call it anymore?

Reply:
No it is not used. Removed it.